research/delf/delf/python/training/README.md

```diff
@@ -118,11 +118,116 @@ files can take up to 12 hours and up to 500 GB of space disk.*
 
 ## Running the Training
 
-Assuming the TFRecord files were generated in the `gldv2_dataset/tfrecord/` directory, running 
-the following command should start training a model:
+For the training to converge faster, it is possible to initialize the ResNet backbone with the
+weights of a pretrained ImageNet model. The ImageNet checkpoint is available at the following
+location: [`http://storage.googleapis.com/delf/resnet50_imagenet_weights.tar.gz`](http://storage.googleapis.com/delf/resnet50_imagenet_weights.tar.gz).
+To download and unpack it run the following commands on a Linux box:
+```
+curl -Os http://storage.googleapis.com/delf/resnet50_imagenet_weights.tar.gz
+tar -xzvf resnet50_imagenet_weights.tar.gz
+```
 
+Assuming the TFRecord files were generated in the `gldv2_dataset/tfrecord/` directory, running 
+the following command should start training a model and output the results in the `gldv2_training`
+directory:
 ```
-python3 train.py \
+python train.py \
```

Comment:
python --> python3

(just to make sure users are using the right one, I think it's probably better to be explicit)

Reply:
Changed to python3